release-notes.md

```diff
@@ -1,3 +1,11 @@
+# Rigel-18.20.0
+## Static Content Conversion Profile For Webex
+* Issue Type: Task
+* Issue ID: PLAT-23958
+
```

Comment:
add section

### Configuration ###  
* Add webex static conversion profiles configuration to runtime_config confmap in order to activate it